cmp/panel/impl/PanelHeader.js

```diff
@@ -25,21 +25,15 @@ class PanelHeader extends Component {
         return hbox({
             cls: 'xh-panel-header',
             items: [
-                icon ? icon : null,
+                icon || null,
                 title ? box({
+                    cls: 'xh-panel-header-title',
                     flex: 1,
-                    items: [
-                        box({
-                            flex: 1,
-                            cls: 'xh-header-title',
-                            item: title
-                        }),
-                        ...headerItems
-                    ]
-                }) : null
+                    item: title
+                }) : null,
+                ...headerItems
```

Comment:
I made this change here because I didn't want the headerItems to go away if you didn't specify a string title but I didn't test it, nor did I have a chance yet to look at e.g. how do things behave w.r.t. wrapping which is something we should consider.